src/test/resources/cliointegrationtest/clio_test.py

```diff
@@ -48,8 +54,51 @@ def test_health():
     assert js['elasticsearch']['dataNodes'] > 1
 
 
+def test_readgroups_mapping():
+    r = requests.get(elasticsearch_http_uri + '/readgroups/_mapping/default')
+    js = r.json()
+    mappings = js['readgroups']['mappings']['default']
+    assert mappings['dynamic'] == 'false'
+    fields = mappings['properties']
+    assert fields['analysis_type']['type'] == 'keyword'
```

Comment:
Could this be made data driven, e.g. using a structure that maps 'keyword' to all the keyword fields 'date' to the date fields, etc. Then all these asserts could be replaced with a loop (or nested loops) and a single assert